lair_of_xaldern/load_logic.py

```
@@ -0,0 +1,42 @@
+        JOIN enemies e ON re.enemy_id = e.id
+        WHERE re.room_id = ? AND re.is_alive = 1        
+"""
+
+    cursor.execute(sql_query, (room_id,))
+   
+    return cursor.fetchone()
+
+
+#Test
+if __name__ == "__main__":
+    print(f"--- DEBUG INFO ---")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)